source/AI.cpp

```diff
@@ -3448,7 +3441,8 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player)
 	}
 	else if(autoPilot.Has(Command::JUMP))
 	{
-		bool isNewPress = keyDown.Has(Command::JUMP) || !keyHeld.Has(Command::JUMP);
+		bool isNewPress = !isRequestingJump && activeCommands.Has(Command::JUMP);
+		isRequestingJump = activeCommands.Has(Command::JUMP);
```

Comment:
It'd be nice if we could avoid needing a new class member variable just to avoid repeatedly playing the fail sound.

Reply:
Agreed, and I apparently also solved this by using WAIT and sending the JUMP command only once as active command. So I could remove this code and still have the audio and error-message played/shown only once.